code/modules/vending/vending.dm

```diff
@@ -408,32 +408,66 @@
 		product_list += new/datum/data/vending_product(/obj/item/device/flash/turbo, rand(1, 6), hidden=1)
 		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/a38, rand(1, 2), hidden=1) // Obtaining a backpack full of lethal ammo required no effort whatsoever, hence why nobody ordered AP speedloaders from the Syndicate (Convair880).
 
-/obj/machinery/vending/security_ammo
+/obj/machinery/vending/security_ammo //ass jam time yes
+#if ASS_JAM
+	name = "Kinetitech"
+	desc = "A restricted weapon vendor, banned by the Space Geneva Convention in 2036 for being a 'warcrime'. Where the hell did the Head of Security find this?"
+	mats = 6669 //Yes its not traitor restricted, but good luck getting more of these. If you manage to get this, you deserve it
+#else
 	name = "AmmoTech"
 	desc = "A restricted ammunition vendor."
+#endif
 	icon_state = "sec"
 	icon_panel = "standard-panel"
 	icon_deny = "sec-deny"
 	req_access_txt = "37"
 	acceptcard = 0
+#if !ASS_JAM
 	is_syndicate = 1 // okay enough piles of spes ammo for any mechanic
-
+#endif
 	lr = 1
 	lg = 0.8
 	lb = 0.9
 
 	create_products()
 		..()
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/abg, 6)
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/a38, 2)
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/a38/stun, 3)
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/flare, 3)
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/smoke, 3)
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/tranq_darts, 3)
-		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/tranq_darts/anti_mutant, 3)
+		product_list += new/datum/data/vending_product(/obj/item/ammo/bullets/abg, 6 * (ASS_JAM * 20))//on ass jam theres 20 times as much ammo in the ammotech
```

Comment:
Again, better to wrap in the #if/else/endif. Duplicate code for the sake of readability - making it explicitly clear what is and is not part of ass_jam content